assets/images/blue_tile.svg

```diff
@@ -0,0 +1,26 @@
+<svg width="346" height="221" viewBox="0 0 346 221" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The code is using `distribution_blue.svg`. Can this one be removed? The same comment applies to other 2 colors.